setup.py

```diff
@@ -1,21 +1,49 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+
+
+__version__ = '0.1.0-dev'

```

Comment:
I'd prefer the version to be in just one place
see one of many ways to do it here https://github.com/kwikteam/phy/blob/master/setup.py#L38